pkg/reading/models.go

```diff
@@ -0,0 +1,12 @@
+package reading
+
+import "github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/assets"
+
+type VersionStatus struct {
+	Status string `json:"status"`
+}
+
+type InResponse struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type InResponse should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)